app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +239,19 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      <br/>
+
+      {@state.workflows.length > 0 && (
+        <Paginator
+          className="talk"
+          page={@state.workflows[0]._meta.workflows.page}
+          pageCount={@state.workflows[0]._meta.workflows.page_count}
+        />
+      )}
+
+      <hr/>
```

Comment:
Just a minor style thing, but could you put a space between the tag name and its close slash?: `<hr />`